.github/workflows/e2e-nextjs.yml

```diff
@@ -0,0 +1,83 @@
+name: Next.js Integration test
+
+on:
+  pull_request:
+    branches: [main]
+    # Run on release PRs only
+    paths: ['CHANGELOG.md']
+
+jobs:
+  integration:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        bundler: ['default', 'esbuild']
+    steps:
+      - name: Checking out zip-it-and-ship-it
+        uses: actions/checkout@v2
+        with:
+          path: zip-it-and-ship-it
+      - name: Checking out netlify-plugin-nextjs
+        uses: actions/checkout@v2
+        with:
+          repository: netlify/netlify-plugin-nextjs
+          path: netlify-plugin-nextjs
+      - name: Checking out netlify-cli
+        uses: actions/checkout@v2
+        with:
+          repository: netlify/cli
+          path: netlify-cli
+      - name: Installing Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
```

Comment:
Do we want to use the [latest](https://github.com/netlify/zip-it-and-ship-it/blob/c7b62ca7080b6a1bbe77483fde32e9bcde326855/.github/workflows/workflow.yml#L16) Node.js version?

Reply:
Yes! I'll change.

Reply:
Done in https://github.com/netlify/zip-it-and-ship-it/pull/554/commits/8b0c35fa76d232e6a37772a422ee449d77555021.